crew

```diff
@@ -619,6 +616,13 @@ def post_install (dest_dir)
     puts 'Performing post-install...'
     @pkg.postinstall
   end
+  # Run libtool warning commands
+  @warning = 'libtool: warning: remember to run '
+      @out = "#{CREW_BREW_DIR}/libtool.txt"
+  if File.exist? "#{CREW_BREW_DIR}/libtool.txt" then
+    system "grep -oP \"(?<=#{@warning}\').*?(?=\')\" #{@out} | bash"
+    FileUtils.rm "#{CREW_BREW_DIR}/libtool.txt"
```

Comment:
`FileUtils.rm @out`?  Also, I don't see where `libtool` is actually being executed.  Did I miss something?

Reply:
I think you definitely need to have a place here where libtool is called. But if libtool is called properly, we shouldn't get these error messages in the first place, right?

Reply:
@satmandu I tried to remove `libtool` but the warnings are still here